include/ardpolyfills/Arduino.hxx

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Arduino.hxx
+ *  Copyright 2020 ItJustWorkTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#ifndef SMARTCAR_EMUL_ARDUINO_HXX
+#define SMARTCAR_EMUL_ARDUINO_HXX
+#include <Thread>
+#include <chrono>
+#include <iostream>
+#include <stdint.h>
+
+typedef enum { RIGHTMOSTBIT = 0, LEFTMOSTBIT = 1 } SignificantBit;
+
+typedef enum { HIGH = true, LOW = false } digitalValue;
```

Comment:
Same as above; may use `bool` as underlying type
Also, incorrect naming scheme; types should be PascalCase
```cpp
enum DigitalValue : bool {
  LOW,
  HIGH
};
```